website/pages/api-docs/operator/scheduler.mdx

```diff
@@ -0,0 +1,157 @@
+---
+layout: api
+page_title: Scheduler - Operator - HTTP API
+sidebar_title: Scheduler
+description: |-
+  The /operator/scheduler endpoints provide tools for management of Nomad server scheduler settings.
+---
+
+# Scheduler Operator HTTP API
+
+The `/operator/scheduler` endpoints provide tools for management of Nomad server scheduler settings.
+
+## Read Scheduler Configuration
+
+This endpoint retrieves the latest Scheduler configuration. This API was introduced in
+Nomad 0.9 and currently supports enabling/disabling preemption. More options may be added in
+the future.
+
+| Method | Path                                   | Produces           |
+| ------ | -------------------------------------- | ------------------ |
+| `GET`  | `/v1/operator/scheduler/configuration` | `application/json` |
+
+The table below shows this endpoint's support for
+[blocking queries](/api-docs#blocking-queries) and
+[required ACLs](/api-docs#acls).
+
+| Blocking Queries | ACL Required    |
+| ---------------- | --------------- |
+| `NO`             | `operator:read` |
+
+### Sample Request
+
+```shell-session
+$ curl \
+    https://localhost:4646/v1/operator/scheduler/configuration
+```
+
+### Sample Response
+
+```json
+{
+  "Index": 5,
+  "KnownLeader": true,
+  "LastContact": 0,
+  "SchedulerConfig": {
+    "CreateIndex": 5,
+    "ModifyIndex": 5,
+    "SchedulerAlgorithm": "spread",
+    "PreemptionConfig": {
+      "SystemSchedulerEnabled": true,
+      "BatchSchedulerEnabled": false,
+      "ServiceSchedulerEnabled": false
+    }
+  }
+}
+```
+
+#### Field Reference
+
+- `Index` `(int)` - The `Index` value is the Raft commit index corresponding to this
+  configuration.
+
+- `SchedulerConfig` `(SchedulerConfig)` - The returned `SchedulerConfig` object has configuration
+  settings mentioned below.
+
+  - `SchedulerAlgorithm` `(string: "binpack")` - Specifies whether scheduler binpacks or spreads allocations on available nodes.
+
+  - `PreemptionConfig` `(PreemptionConfig)` - Options to enable preemption for various schedulers.
+
+    - `SystemSchedulerEnabled` `(bool: true)` - Specifies whether preemption for system jobs is enabled. Note that
+      this defaults to true.
+
+    - `BatchSchedulerEnabled` `(bool: false)` - Specifies whether preemption for batch jobs is enabled. Note that
+      this defaults to false and must be explicitly enabled.
+
+    - `ServiceSchedulerEnabled` `(bool: false)` - Specifies whether preemption for service jobs is enabled. Note that
+      this defaults to false and must be explicitly enabled.
+
+  - `CreateIndex` - The Raft index at which the config was created.
+  - `ModifyIndex` - The Raft index at which the config was modified.
```

Comment:
```suggestion
  - `ModifyIndex` - The Raft commit index at which the config was modified.
```

Reply:
Same as some of the otheres, but Raft index seems to be used widely in the docs / internal comments but raft commit index is not, hesitant to change

Reply:
I noticed inconsistencies within this doc; sometimes it's Raft commit index, sometimes just index. I'm fine with either, but consistency is nice.

Reply:
:+1: converging on `Raft index`